### DISCUSSION TITLE
Interpretation  of MQ-2 vaules

### DISCUSSION THREAD

**Discussion by AlinaSmarthome** [arendst/Tasmota#19563] [Q&A]
Hello

i have installed Tasmota 13.1.0 und linked a MQ-2 gas sensor to a WEMOS D1 (5V, GND an A0) but the values lookes sth strange.
I only get values in the range 0.1 to 0.4 "ppm" as told in Tasmota WEB GUI. I have read some docs and have looked to the source code of that part but from my understanding the value should at least 100ppm

Any idea/ understandings?

Thx
Alina

**Comment by sfromis** [comment]
Did you make sure to use `AdcParam 10` to select parameters?
https://tasmota.github.io/docs/ADC/#commands

**Comment by AlinaSmarthome** [comment]
Hello thanks for the hint. no i haven't set up

To be right here: for MQ-2 it would be:

AdcParam 10, 2.00, 574.25, -2.22, 9.83

IsN't it?

**Reply by sfromis** [reply]
Not using MQ sensors, IDK, but it seems to match the docs. Are you using those values? If not, try following the docs.

**Comment by AlinaSmarthome** [comment]
Well done Sir, please ask us only if you already knows the answer
